### PR TITLE
Added phantoms and popups library

### PR DESCRIPTION
**Pull request by DamnWidget**
Added support for themeable ST3 minihtml popups and phantoms with support for code syntax highlight and click events hooking.

Popup example:
screenshot_20161130_190544

Phantom example:
screenshot_20161124_211144

We will need those to add documentation/hinting support. Both phantoms and popups support title plus body or just body with HTML or MD syntax and customizable CSS. The widgets contents are wrapped into the class `kite-popup` and `kite-phantom` so we can override theme specific styles if we need to.

### CODE REVIEW

File: dependencies.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (alexflint) [inline dependencies.json]: does this cause sublimetext to automatically install these from pip, or does the user have to trigger that manually?

File: lib/popups.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+            if not callable(handler):
+                raise RuntimeError(
+                    'tried to add non callable object: {0} '
+                    'into {1} handlers registry as {2}!'.format(
+                        handler, self, url
+                    )
+                )
+
+            self.navigation_handlers['url'] = handler
```
Comment (alexflint) [inline lib/popups.py]: not sure if this is correct but shouldn't this be `url` not `'url'`?

File: dependencies.json (hunk 1 above)
Comment (DamnWidget) [inline dependencies.json]: Yes, it will make Package Control to install it automatically when ST3 starts.

File: lib/popups.py (hunk 2 above)
Comment (DamnWidget) [inline lib/popups.py]: You are right

**Pull request merged**